### ISSUE TITLE
🧪 Mad Science Project Idea: L2s for Non-EVM L1s

### ISSUE THREAD

**Issue by smartcontracts** [ethereum-optimism/ecosystem-contributions#22]
**What is an Mad Science Project Idea?**

A Mad Science Project Idea is an idea for a project within the Optimism ecosystem that may not have clear value but is generally cool as heck. Mad Science Project Ideas make great hackathon projects and tend to be a great way to explore Optimism, OP Mainnet, and the OP Stack.

---

**Summary**

[The OP Stack](https://stack.optimism.io/) makes it possible to easily spin up new rollups. Why not create a rollup for an L1 that isn't Ethereum? Maybe a rollup that settles to Solana, Cardano, or Bitcoin? You get the idea.

**Possible Features**

- Create an OP Stack chain that settles to a non-EVM chain.

**Resources**

- [The OP Stack](https://stack.optimism.io/)

**Comment by bstr156** [comment]
<div align="center">
  <br />
  <br />
  <a href="https://rollux.com"><img alt="Rollux" src="https://raw.githubusercontent.com/SYS-Labs/brand-kits/main/rollux/SVG/rollux_logo.svg" width=100></a>
</div>

### Mad science indeed!

Happy to present this to you all :)
Please let us know if you have any questions!!!  Would be fun participate in a DA discussion on OP Radio.

[Rollux](https://rollux.com) is an Optimism fork refactored for a new Non-EVM DA, and is now mainnet and settling on the Syscoin layer 1 through Syscoin’s PoDA protocol (Proof of Data Availability). PoDA is generalized DA that serves the OPStack community as well as other Layer 2s including Validity-based solutions. A unique take, and the Rollux fork is first to present it in a production environment! I think you will find Rollux's use of PoDA very interesting to consider as a component in future “Multiple-DA”, and as a sample case for better-generalizing DA in Superchain (a project Syscoin Foundation is interested in).

Rollux mainnet:
https://chainlist.org/?search=rollux

Code diff between Optimism and Rollux DA: https://github.com/SYS-Labs/rollux/commit/25a4c9410ddae31ff7195f67495491f71e684e03

An overview of PoDA's architecture:
https://docs.rollux.com/docs/sys/PoDA/#optimistic-rollup-with-poda-proof-of-data-availability

### WHY?

**PoDA benefits L2s in areas of cost, security and resilience, with tradeoffs that are philosophically interesting!**

**Cost:** The most immediate noticeable benefit comes in the form of very low cost L1 settlement with a DA-tailored fee market. This can be seen by comparing the Optimism sequencer costs of using Ethereum with those of Rollux using Syscoin. While it appears Optimism Bedrock incurs ~1 ETH per hour (rough estimate) using Ethereum calldata, Rollux Bedrock typically spends 0.0001 SYS per hour for L1 DA. From the standpoint of an L2 operator, this means by utilizing Syscoin the L2 fees remain incredibly easy to subsidize even under future burdens of adoption.

**Security & Resilience:** The security of PoDA is a factor of PoDA’s design as well as Syscoin’s.

- **PoDA** is different from EIP-4844 in a few very important ways. In our design, we separate DA into two distinct concerns; archiving and proving. PoDA utilizes hybrid storage, both onchain (succinct proof of payload, on UTXO) and offchain (full data payload) using IPFS (Filecoin + Lighthouse) and/or cloud etc. PoDA archiving is handled by full nodes operating in an open-participation network, meaning any node can serve as a PoDA archiver. Due to efficient use of L1 real-estate combined with offchain payload storage, (1) sharding is not needed - only one honest node is necessary for security guarantees which leans towards better resilience against censorship attacks, (2) Efficient keccak is viable in place of more intensive KZG required by proto-danksharding, (3) L1 nodes can remain lightweight and easy to sync from scratch.
- **Syscoin**, in brief, serves as the supporting chain of PoDA, which means OPStack inherits Bitcoin merge-mined PoW, combined with a multi-quorum finality layer which practically eliminates typical PoW risks of 51% attacks and selfish mining while the larger design keeps the Nakamoto consensus intact from beneath. Further, it has its own EIP-1559-based economy in SYS instead of interfering with BTC economics or requiring users to burn BTC on gas while still getting the best security attributes the Bitcoin network can offer. Much more information is available. Suffice to say, Syscoin’s modular L1 is dual-chain (UTXO and EVM side-by-side) and offers comparable (or in some ways better) guarantees and capabilities today as what Ethereum plans to provide once proto-danksharding is added… while retaining PoW at the base for better resilience against black swans.

In summary, you get somewhat comparable cost/performance benefits from PoDA as you might from using Validiums or non-L1 DA solutions out there.  HOWEVER, you also retain the benefits of L1 settlement, on a well-decentralized and resilient L1 at that.

Cheers!

Interested in latest happenings with Rollux & Syscoin? https://x.com/RolluxL2  and  https://x.com/syscoin

**Comment by bstr156** [comment]
Related to PR https://github.com/ethereum-optimism/stack-docs/pull/25 (pending review at present time)

**Comment by AlexGasHawk** [comment]
Hi I work with GasHawk.io and am interested in building this for the OP ecosystem, is this project still accepting applications?
